include/re_types.h

```diff
@@ -81,6 +81,8 @@ typedef SSIZE_T ssize_t;
 /** Defines a soft breakpoint */
 #if (defined(__i386__) || defined(__x86_64__))
 #define RE_BREAKPOINT __asm__("int $0x03")
+#elif defined(__aarch64__)
+#define RE_BREAKPOINT __builtin_debugtrap()
```

Comment:
I think we can make this more generic with:

```c
#elif defined(__has_builtin) &&  __has_builtin(____builtin_debugtrap)
#define RE_BREAKPOINT __builtin_debugtrap()
```

Reply:
Yes, that's much better. I've changed it accordingly.

Reply:
Looks like we need a modification for MSVC
```c
#elif !defined(MSVC) && \
	defined(__has_builtin) && __has_builtin(__builtin_debugtrap)
```

Reply:
I'll have an eye on the validation run

Reply:
Looks good